#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("global_frame_id", str_t, 0, "frame_id of global cooridnates", "odom")
gen.add("horizontal_angular_threshold", double_t, 0, "Horizontal Angular Threshold",  0.1, 0, 1.57)
gen.add("vertical_angular_threshold", double_t, 0, "Vertical Angular Threshold",  0.1, 0, 1.57)


exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "PlaneReasoner"))
